files/ui/graphicspage.ui

```diff
@@ -146,6 +146,70 @@
      </layout>
     </widget>
    </item>
+   <item>
+    <widget class="QGroupBox" name="gridGroupBox">
+     <property name="title">
+      <string>Distant Terrain</string>
+     </property>
+     <layout class="QGridLayout" name="gridLayout_4" columnstretch="1,0">
+      <item row="5" column="0">
+       <widget class="QCheckBox" name="distantTerrainCheckBox">
+        <property name="text">
+         <string>Enable</string>
+        </property>
+       </widget>
+      </item>
+      <item row="7" column="0">
+       <widget class="QLabel" name="smallFeatureCullingLabel">
+        <property name="text">
+         <string>Small Feature Culling</string>
+        </property>
+        <property name="alignment">
+         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
+        </property>
+       </widget>
+      </item>
+      <item row="7" column="1">
+       <layout class="QGridLayout" name="smallFeatureGridLayout">
+        <item row="0" column="1">
+         <layout class="QHBoxLayout" name="smallFeatureBoxLayout"/>
+        </item>
+        <item row="0" column="0">
+         <widget class="QCheckBox" name="smallFeatureCullingCheckBox">
+          <property name="text">
+           <string>Enable</string>
+          </property>
+         </widget>
+        </item>
+        <item row="0" column="1">
```

Comment:
We have two items at `row="0" column="1"`. Those should have separate numbers.

Reply:
The second item is under the layout @173, and the first one is outside it.